CogniteSdk/test/fsharp/Alpha/SimulatorRoutines.fs

```diff
@@ -396,8 +396,8 @@ let ``Create simulator routine revisions is Ok`` () =
                     Configuration =
                         SimulatorRoutineRevisionConfiguration(
                             Schedule = SimulatorRoutineRevisionSchedule(Enabled = false),
-                            LogicalCheck = SimulatorRoutineRevisionLogicalCheck(Enabled = false),
-                            SteadyStateDetection = SimulatorRoutineRevisionSteadyStateDetection(Enabled = false),
+                            LogicalCheck = [SimulatorRoutineRevisionLogicalCheck(Enabled = false)],
+                            SteadyStateDetection = [SimulatorRoutineRevisionSteadyStateDetection(Enabled = false)],
```

Comment:
enabled false is not allowed in the API anymore

Reply:
or maybe it is, but then the other fields are supposed to be available, so either to just pass an empty array

Reply:
good catch, fixing it